website/README.md

```diff
@@ -303,4 +294,25 @@ Auto fixing `JS` lint issues
 
 Running `Webpack` build (production)
 
-    inv run-build
\ No newline at end of file
+    inv run-build
```

Comment:
Docker instructions were moved to down here @Baalmart

Reply:
Alright, all makes sense now. Thanks @FaithDaka